packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.jsx

```diff
@@ -69,6 +89,12 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
         setDialogVisibility(!is_dialog_open);
     };
 
+    const onSearchBlur = () => {
+        RudderStack.track('ce_trade_types_form', {
+            search_string: search_query,
```

Comment:
Should this be like:
```
 RudderStack.track('ce_trade_types_form', {
            action: 'search',
            search_string: search_query,
})

Reply:
Ah yes, seems like I missed the action 👍🏻

Reply:
Thanks for pointing out. This is fixed.